packages/preview/paddling-tongji-thesis/0.1.1/init-files/main.typ

```diff
@@ -0,0 +1,61 @@
+#import "./tongjithesis.typ": *
```

Comment:
If you move `tongjithesis.typ` as I suggested, you will have to replace this import with `#import "@preview/paddling-tongji-thesis:0.1.1: *"`. This is because only the files in `init-files` will be copied when creating a project, so you have to use the package reference from the package index, not a local file path.